### ISSUE TITLE
Error on collection creation

### ISSUE THREAD

**Issue by AurelienKun** [appwrite/appwrite#2606]
I've just installed Appwrite with docker and logged in, created a new project.
Right into the UI (from http://localhost/console/database?project=my.custom.project), when trying to add manually a collection I get an error. 
Here are the logs from appwrite container :
```
[Error] Message: Incorrect table name '_project_my.custom.project_collection_61e032edc1928e8ffa53'
[Error] File: @swoole-src/library/core/Database/PDOStatementProxy.php
[Error] Line: 64
[Error] Type: PDOException
```
I also found those logs in Appwrite-worker-database :
```
Appwrite database worker v1 has started
[notice] Starting worker b30d26364924:7:v1-database
Fatal error: Uncaught RedisException: socket error on read socket in /usr/src/code/vendor/colinmollenhour/credis/Client.php:1171

Stack trace:
#0 /usr/src/code/vendor/colinmollenhour/credis/Client.php(1171): Redis->ping()
#1 /usr/src/code/vendor/resque/php-resque/lib/Resque/Redis.php(265): Credis_Client->__call('ping', Array)
#2 /usr/src/code/vendor/resque/php-resque/lib/Resque/Worker.php(180): Resque_Redis->__call('ping', Array)
#3 /usr/src/code/vendor/resque/php-resque/bin/resque(185): Resque_Worker->work('0.1', false)

}
Next CredisException: socket error on read socket in /usr/src/code/vendor/colinmollenhour/credis/Client.php:1189

Stack trace:
#0 /usr/src/code/vendor/resque/php-resque/lib/Resque/Redis.php(265): Credis_Client->__call('ping', Array)
#1 /usr/src/code/vendor/resque/php-resque/lib/Resque/Worker.php(180): Resque_Redis->__call('ping', Array)
#2 /usr/src/code/vendor/resque/php-resque/bin/resque(185): Resque_Worker->work('0.1', false)

}
Next Resque_RedisException: Error communicating with Redis: socket error on read socket in /usr/src/code/vendor/resque/php-resque/lib/Resque/Redis.php:268

Stack trace:
#0 /usr/src/code/vendor/resque/php-resque/lib/Resque/Worker.php(180): Resque_Redis->__call('ping', Array)
#1 /usr/src/code/vendor/resque/php-resque/bin/resque(185): Resque_Worker->work('0.1', false)

}
  thrown in /usr/src/code/vendor/resque/php-resque/lib/Resque/Redis.php on line 268
```
---
Appwrite Version: 0.12.0.200

**Comment by eldadfux** [comment]
Thanks for reporting the issue, seems like the project ID contains `.` which breaks the query. We probably need to wrap the internal query with quotes. For now changing the project custom ID to something with no dots should do the trick.

**Comment by kodumbeats** [comment]
@AurelienKun We aren't able to recreate this issue - have you run into this again? If so, please share your steps <3

**Comment by AurelienKun** [comment]
@kodumbeats  
I've just created a new test project and went immediately to collection, and I could not reproduce the problem. I have only the issue on the very first project I created (with a custom id containing dots, à la android). I created the project just after installing everything via docker by copy/pasting the commands from the readme. I can still produce the error.

Tell me what can I do to help you debugging this issue.
